letsencrypt/cli.py

```diff
@@ -921,7 +929,13 @@ def _create_subparsers(helpful):
     helpful.add_group("revoke", description="Options for revocation of certs")
     helpful.add_group("rollback", description="Options for reverting config changes")
     helpful.add_group("plugins", description="Plugin options")
-
+    helpful.add(
+        None, "--user-agent", type=str, default=None,
```

Comment:
Why specify `type=str`? This is the default ([source](https://docs.python.org/2/library/argparse.html#type)).